src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const ageOfMen = [];
+
+  people.filter(({ born, died, sex }) => {
+    const diedCentury = Math.ceil(died / 100);
+    const age = died - born;
+
+    if ((century && century === diedCentury && sex === 'm')
+      || (!century && sex === 'm')) {
```

Comment:
Move century comparing logic to a variable, like 
`
  const centuryCheck = century
    ? century === diedCentury
    : true;
`